storybook/img/logo-alt.svg

```diff
@@ -1,18 +1,18 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 16.0.4, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 width="37px" height="35px" viewBox="0 0 37 35" enable-background="new 0 0 37 35" xml:space="preserve">
-<linearGradient id="SVGID_1_" gradientUnits="userSpaceOnUse" x1="231.8955" y1="-269.5547" x2="231.8955" y2="-301.7199" gradientTransform="matrix(1 0 0 -1 -213.5 -268.5)">
-	<stop  offset="0" style="stop-color:#60EFFF"/>
-	<stop  offset="1" style="stop-color:#1F89C7"/>
-</linearGradient>
-<path fill="url(#SVGID_1_)" d="M36.792,17.992L18.398,0L0,17.992l7.333,10.39l5.595-1.285l5.503,7.737l5.493-7.724l5.533,1.27
-	L36.792,17.992z M17.634,3.336L15.25,25.355l-4.549-6.205L17.634,3.336z M10.202,18.47l-2.833-3.866l9.743-11.833L10.202,18.47z
-	 M14.495,25.562L14.495,25.562l0.265,0.365l-1.52,0.348l-3.507-4.93l0.626-1.424L14.495,25.562z M18.396,3.044l2.395,23.345
-	l-2.4,3.288L16,26.389L18.396,3.044z M22.296,25.562l4.181-5.702l0.654,1.485l-3.517,4.944l-1.58-0.362l0.264-0.363L22.296,25.562z
-	 M21.542,25.355L19.189,3.318l6.946,15.77L21.542,25.355z M19.744,2.767l9.679,11.837l-2.787,3.802L19.744,2.767z M7.647,27.56
-	l-6.692-9.48L14.051,5.278l-7.605,9.302l3.417,4.661l-0.964,2.188l3.572,5.023L7.647,27.56z M18.432,33.573l-4.733-6.653
-	l1.528-0.352l3.162,4.351l3.173-4.351l1.592,0.367L18.432,33.573z M24.384,26.466l3.581-5.037l-0.99-2.248l3.372-4.6l-7.605-9.302
-	l13.095,12.801l-6.692,9.48L24.384,26.466z"/>
-</svg>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@redallen is the bot supposed to modify the whitespace of these files?

Reply:
It didn't modify it. Titani did in f15cb169978f10ccbd06edc178b6735438469310.

Reply:
Titani's changes only affected 2 snapshot files from what I can see. The first commit by the bot appears to have the img files modified